met-api/src/met_api/services/email_verification_service.py

```diff
@@ -53,11 +62,30 @@ def verify(self, verification_token, survey_id):
         return db_email_verification
 
     @staticmethod
-    def send_verification_email(email_verification: EmailVerificationSchema) -> DefaultMethodResult:
-        """Send an verification email."""
-        print(email_verification)
-        # TODO: send email.
-        return DefaultMethodResult(True, '', 0)
+    def _send_verification_email(email_verification: dict) -> None:
+        """Send an verification email.Throws error if fails."""
+        sender = current_app.config.get('MAIL_FROM_ID')
+        survey_id = email_verification.get('survey_id')
+        email_to = email_verification.get('email_address')
+        survey: SurveyModel = SurveyModel.get_survey(survey_id)
```

Comment:
I think better to use get_open_survey here to make sure it is also open for user.